api/net/Areas/Services/Controllers/ReportController.cs

```diff
@@ -83,16 +83,19 @@ public IActionResult FindById(int id)
     /// Make a request to Elasticsearch for content that matches the specified report 'id' filter.
     /// </summary>
     /// <param name="id"></param>
+    /// <param name="requestorId"></param>
     /// <returns></returns>
     [HttpGet("{id}/content")]
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(Dictionary<string, Elastic.Models.SearchResultModel<API.Areas.Services.Models.Content.ContentModel>>), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType((int)HttpStatusCode.NoContent)]
     [SwaggerOperation(Tags = new[] { "Report" })]
-    public async Task<IActionResult> FindContentForReportIdAsync(int id)
+    public async Task<IActionResult> FindContentForReportIdAsync(int id, int? requestorId)
```

Comment:
Including the requestor allows other users to generate instances from a report they do not own.